storybook/src/admin/error/inlineError/InlineError.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import { InlineError } from "@comet/admin";
+import { Clear, CometColor, InfoFilled, Reload, RemoveFilled, WarningSolid } from "@comet/admin-icons";
+import { Box, Button } from "@mui/material";
+import type { Meta, StoryObj } from "@storybook/react";
+
+type Story = StoryObj<typeof InlineError>;
+
+const config: Meta<typeof InlineError> = {
+    component: InlineError,
```

Comment:
Cool, didn't know you could define stories just like that combined with the comments and `args` below 👍🏼 

A bit odd though, that the `locale` and `theme` props are added in the story's code though 🤔 

<img width="1022" alt="image" src="https://github.com/user-attachments/assets/832e7ffe-1caf-46db-aaa7-b4dc08dc09e9" />

Reply:
that's nice, yeah, I think they call it CSF format or so.

I have also seen the problem with locale and theme, but I think that comes from somewhere different. That's something (in my opinion, out of the scope of this pr) that should be reconfigured in the storybook setup.

Reply:
@manuelblum how should we proceed here?

Reply:
This is out of scope imo, I created a task for this: https://vivid-planet.atlassian.net/browse/COM-1896